_includes/nav/README.md

```diff
@@ -0,0 +1,182 @@
+# Navigation includes
+
+This directory is a collection of Jekyll includes that can be used to
```

Comment:
s collection/set - only b/c collections are a Jekyll "thing" and want to help avoid any confusion.